cpp/oneapi/dal/algo/svm/test/batch.cpp

```diff
@@ -375,6 +375,7 @@ TEMPLATE_LIST_TEST_M(svm_batch_test,
                      "svm can classify linear separable surface with big margin",
                      "[svm][integration][batch][linear]",
                      svm_types) {
+    SKIP_IF(this->get_policy().is_cpu());
```

Comment:
Why this test is skipped on CPU? If it's temporal, corresponding comment should be added, `// TODO: `